utilities_item.js

```diff
@@ -1030,7 +1030,7 @@ var Utilities_Item = {
 
 		let onlyNumbersRe = /^\d*$/;
 		let deweyRe = /^(\d{3})(?:\.(\d+))?(?:\/([a-zA-Z]{3}))?$/;
-		let lcWithClassificationRe = /^[a-zA-Z]{1,3}\d+($|(?=\s*[.\d]))/;
+		let lcWithClassificationRe = /^[a-zA-Z]{1,3}\d+($|(?=\s*(.|\s\d)))/;
```

Comment:
This fixes the recognition and matching of patterns like "M100a", where before "M10" was recognized as the main class and the next "0" as the look-ahead here.